libs/uriparser/Makefile

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (C) 2015-2018 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
```

Comment:
Hmmm, no. Check it here: https://github.com/openwrt/packages/blob/81d53813d63dcbedf1489946af9854070c3b80a0/CONTRIBUTING.md

Reply:
Will fix.